eclipse-che/che-plugin-registry
# chore: use editor's devfile as it is

### What does this PR do?
Don't do extra modifications of the editors declared in **che-editors.yaml** file.
Che-editors.yaml file provides devfiles 2.2.2 that define everything needed for editors, we just need to use them as they are. 

### Screenshot/screencast of this PR

![screenshot-eclipse-che apps ci-ln-pxb49rk-76ef8 aws-2 ci openshift [org-2024](https://issues.redhat.com//browse/org-2024) 01 25-17_57_35](https://github.com/eclipse-che/che-plugin-registry/assets/1271546/19f62565-6594-42c6-9ae3-ca888510c152)

![screenshot-eclipse-che apps ci-ln-pxb49rk-76ef8 aws-2 ci openshift [org-2024](https://issues.redhat.com//browse/org-2024) 01 25-17_54_01](https://github.com/eclipse-che/che-plugin-registry/assets/1271546/c6e9e862-194c-4298-a027-b5597fd9849f)


### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/19954

### How to test this PR?

Current changes don't provide any new functionality, it's just code cleanup of editors' devfiles generation. So, we need to test that everything works as usual.

1. Current changes are available with **quay.io/vsvydenk/che-plugin-registry:no-meta** image. So, need to deploy Che and use this image for the plugin-registry
2.  Try to start workspaces using factory wit different editors:
-  che-idea-latest: https://github.com/svor/web-nodejs-sample/tree/che-idea-latest
- che-code-latest: https://github.com/svor/web-nodejs-sample/tree/che-code-latest
- che-code-insiders: https://github.com/svor/web-nodejs-sample/tree/che-code-insiders
3. All workspaceses should start with the editor.

**Another test case:**
1. Update CheCluster CR to use **quay.io/vsvydenk/che-devfile-registry:new-editors** image for devfile registry component. This image provides generated samples that are used new editors
2. Now open dashboard and try to start a workspace from the samples
3. Everything should work as usual

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
